lib/pacemaker/pcmk_sched_instances.c

```diff
@@ -725,31 +725,32 @@ multi_update_interleave_actions(pe_action_t *first, pe_action_t *then,
             const pe_resource_t *then_rsc = NULL;
 
             first_rsc = pcmk__get_rsc_in_container(first_child, node);
-            if ((first_rsc != NULL) && strstr(first->task, "stop")) {
-                /* Except for 'stopped' we should be looking at the
-                 * in-container resource, actions for the child will
-                 * happen later and are therefor more likely to align
-                 * with the user's intent.
+            if ((first_rsc != NULL)
+                && pcmk__str_any_of(first->task, CRMD_ACTION_STOP,
```

Comment:
Can we use `first_task` here if we're going to use it later, or is this different?

It also looks like `get_complex_task()` (called by `clone_child_action(first)`) has already converted `CRMD_ACTION_STOPPED` to `CRMD_ACTION_STOP` for `first_task`, if we want to take advantage of that. Being explicit is nice too though.

Reply:
The later work will make this easier to follow